dropwizard-api/src/main/java/org/coner/api/entity/CompetitionGroup.java

```diff
@@ -10,7 +10,7 @@
 import java.math.BigDecimal;
 
 /**
- * REST API entity representing a Competition Group
+ * REST API entity representing a Competition Group.
```

Comment:
Is this failing for you when you run checkstyle? Because for me it isn't nor is it failing on travis. Do you maybe have an uncommitted change sitting in your checkstyle config?